### PR TITLE
Postgres variant logos

### PR DESCRIPTION
**Pull request by plcplc**
This PR adds ndc-hub entries to the following Postgres (and variants) offerings:

* AWS Aurora
* Citus
* CockroachDB
* Neon
* Google Spanner
* YugabyteDB

### CODE REVIEW

File: registry/aurora/metadata.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "is_verified": true,
+  "is_hosted_by_hasura": true,
+  "source_code": {
+    "is_open_source": true,
+    "repository": "https://github.com/hasura/ndc-postgres",
+    "version": [
+      {
+        "tag": "v0.2.0",
+        "hash": "98801634b0e1396c933188eef88178952f412a8c",
```
Comment (soupi) [inline registry/aurora/metadata.json:25]: Perhaps we should wait a bit until we release v0.4.0, and then bump the tag and the hash for all connectors (including postgres)?

**Comment by TheInnerLight** [comment]
Should we add a list of items to the READMEs to show which databases are supported? e.g.

### Supported Variants

- [x] Vanilla Postgres
- [x] AWS Aurora
- [x] Citus
- [x] CockroachDB
- [x] Neon
- [x] Google Spanner
- [x] YugabyteDB

**Comment by plcplc** [comment]
> Should we add a list of items to the READMEs to show which databases are supported?

I think it would definitely be appropriate to update the top-level README.md, perhaps just having it reference the registry folder rather than essentially listing it in prose.

File: registry/aurora/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+`ndc-postgres` provides a Hasura Data Connector to the PostgreSQL database,
+which can expose and run GraphQL queries via the Hasura v3 Project.
+
+
+- [PostgreSQL Connector information in the Hasura Connectors directory](https://hasura.io/connectors/postgres)
+- [GitHub repository](https://github.com/hasura/ndc-postgres)
+
+The connector implements the [NDC Specification](https://hasura.github.io/ndc-spec/overview.html),
+but does not currently support mutations, column relationship arguments in queries, functions or procedures.
```
Comment (soupi) [inline registry/aurora/README.md:11]:
```suggestion
but does not currently support column relationship arguments in queries, or functions.
```

File: registry/aurora/metadata.json (hunk 1 above)
Comment (soupi) [reply registry/aurora/metadata.json:25]: I still see v0.2.0 here and in other places (latest_version). could you please find and replace 0.2.0 to 0.4.0?

File: registry/cockroach/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+## Overview
+
+`ndc-postgres` provides a Hasura Data Connector to the PostgreSQL database,
+which can expose and run GraphQL queries via the Hasura v3 Project.
+
+- [PostgreSQL Connector information in the Hasura Connectors directory](https://hasura.io/connectors/postgres)
+- [GitHub repository](https://github.com/hasura/ndc-postgres)
+
+The connector implements the [NDC Specification](https://hasura.github.io/ndc-spec/overview.html),
+but does not currently support mutations, column relationship arguments in queries, functions or procedures.
```
Comment (soupi) [inline registry/cockroach/README.md:10]:
```suggestion
but does not currently support column relationship arguments in queries, or functions.
```

File: registry/neon/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+## Overview
+
+`ndc-postgres` provides a Hasura Data Connector to the PostgreSQL database,
+which can expose and run GraphQL queries via the Hasura v3 Project.
+
+- [PostgreSQL Connector information in the Hasura Connectors directory](https://hasura.io/connectors/postgres)
+- [GitHub repository](https://github.com/hasura/ndc-postgres)
+
+The connector implements the [NDC Specification](https://hasura.github.io/ndc-spec/overview.html),
+but does not currently support mutations, column relationship arguments in queries, functions or procedures.
```
Comment (soupi) [inline registry/neon/README.md:10]:
```suggestion
but does not currently support column relationship arguments in queries, or functions.
```

File: registry/aurora/metadata.json (hunk 1 above)
Comment (soupi) [reply registry/aurora/metadata.json:25]: Also the hash seems to be different in a few entries.

File: registry/spanner/README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+## Overview
+
+`ndc-postgres` provides a Hasura Data Connector to the PostgreSQL database,
+which can expose and run GraphQL queries via the Hasura v3 Project.
+
+- [PostgreSQL Connector information in the Hasura Connectors directory](https://hasura.io/connectors/postgres)
+- [GitHub repository](https://github.com/hasura/ndc-postgres)
+
+The connector implements the [NDC Specification](https://hasura.github.io/ndc-spec/overview.html),
+but does not currently support mutations, column relationship arguments in queries, functions or procedures.
```
Comment (soupi) [inline registry/spanner/README.md:10]:
```suggestion
but does not currently support column relationship arguments in queries, or functions.
```

File: registry/yugabyte/README.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+## Overview
+
+`ndc-postgres` provides a Hasura Data Connector to the PostgreSQL database,
+which can expose and run GraphQL queries via the Hasura v3 Project.
+
+- [PostgreSQL Connector information in the Hasura Connectors directory](https://hasura.io/connectors/postgres)
+- [GitHub repository](https://github.com/hasura/ndc-postgres)
+
+The connector implements the [NDC Specification](https://hasura.github.io/ndc-spec/overview.html),
+but does not currently support mutations, column relationship arguments in queries, functions or procedures.
```
Comment (soupi) [inline registry/yugabyte/README.md:10]:
```suggestion
but does not currently support column relationship arguments in queries, or functions.
```

File: registry/aurora/metadata.json (hunk 1 above)
Comment (plcplc) [reply registry/aurora/metadata.json]: Thanks for catching this. I had forgotten the aurora connector!

I've also updated the `overview.latest_version` from `v0.2.0` to `v0.4.0`.

**Review by soupi — APPROVED** [review]
Thanks!

**Pull request merged**